### PR TITLE
[WIP] Cleanup LaunchTemplate task

### PR DESCRIPTION
**Pull request by hakman**
Cleanup after merging https://github.com/kubernetes/kops/pull/10937.

### CODE REVIEW

**Review by rifelpet — COMMENTED** [review]
for the record this looks good to me other than the lingering logging statements

File: pkg/model/awsmodel/autoscalinggroup.go [hunk 1]
```diff
@@ -92,144 +93,70 @@ func (b *AutoscalingGroupModelBuilder) Build(c *fi.ModelBuilderContext) error {
...
-		} else {
-			lt.RootVolumeThroughput = fi.Int64(int64(fi.Int32Value(ig.Spec.RootVolumeThroughput)))
-		}
+		CPUCredits:                   fi.String(""),
+		HTTPPutResponseHopLimit:      fi.Int64(DefaultHTTPPutResponseHopLimit),
+		HTTPTokens:                   fi.String(DefaultHTTPTokens),
+		ImageID:                      fi.String(ig.Spec.Image),
+		InstanceInterruptionBehavior: ig.Spec.InstanceInterruptionBehavior,
+		InstanceMonitoring:           ig.Spec.DetailedInstanceMonitoring,
+		InstanceType:                 fi.String(ig.Spec.MachineType),
```
Comment (johngmyers) [inline pkg/model/awsmodel/autoscalinggroup.go:106]: This is a semantic change.

**Review by johngmyers — COMMENTED** [review]
This is extremely difficult to review. Next time it would be better if it were a series of commits, where for each commit it would be simple to determine whether or not the change is a semantic difference.

**Comment by johngmyers** [comment]
By the way, I haven't gotten far in my review due to the difficulty and the likely need to restart at square zero if there is a subsequent rebase.

**Comment by johngmyers** [comment]
Would it be possible to split this up into a sequence of smaller, easier to review steps? This commit is just too big.

**Comment by hakman** [comment]
> Would it be possible to split this up into a sequence of smaller, easier to review steps? This commit is just too big.

Totally agree @johngmyers. Thanks!

**Pull request closed without merging**